### PR TITLE
Update blacklist/whitelist to blockList/allowList

### PR DESCRIPTION
**Pull request by gtrufitt**
## References:

https://tools.ietf.org/id/draft-knodel-terminology-00.html

https://monzo.com/tone-of-voice/#open

https://medium.com/@bronAtNuna/language-matters-7fced5d78f8d

https://www.computerweekly.com/news/252482522/NCSC-tackles-unconscious-bias-in-security-terminology

### CODE REVIEW

File: article/app/services/dotcomponents/ArticlePicker.scala [hunk 1]
```diff
@@ -153,13 +153,13 @@ object ArticlePicker {
       ("isNotAMP", ArticlePageChecks.isNotAMP(request)),
       ("isNotPaidContent", ArticlePageChecks.isNotPaidContent(page)),
       ("isSupportedTone", ArticlePageChecks.isSupportedTone(page)),
-      ("isNotBlackListed", ArticlePageChecks.isNotBlackListed(page)),
+      ("isNotDenyListed", ArticlePageChecks.isNotDenyListed(page)),
```
Comment (tjmw) [inline article/app/services/dotcomponents/ArticlePicker.scala:156]: Super minor but I wonder if `isNotInDenyList` would read slightly better here? It's also perhaps more consistent with `isInAllowList`.

**Review by tjmw — APPROVED** [review]
:100: Looks great. I left one minor comment/suggestion to take or leave.

File: article/app/services/dotcomponents/ArticlePicker.scala (hunk 1 above)
Comment (gtrufitt) [reply article/app/services/dotcomponents/ArticlePicker.scala]: 👍

Comment (sndrs) [reply article/app/services/dotcomponents/ArticlePicker.scala]: how about `blocklist`? sounds more natural to me (and probably actually clearer than `blacklist` too)

Comment (gtrufitt) [reply article/app/services/dotcomponents/ArticlePicker.scala]: +1

**Pull request merged**